spec/lang/syntax.md

```diff
@@ -360,9 +360,16 @@ pub enum Terminator {
         /// The block to jump to when this call returns.
         /// If `None`, UB will be raised when the function returns.
         next_block: Option<BbName>,
+        /// The block to jump to when this call unwinds.
+        /// If `None`, UB will be raised when the function unwinds.
```

Comment:
```suggestion
        /// If `None`, UB will be raised when the function unwinds.
        /// This comes with a well-formedness requirement: if the current block is a regular block,
        /// `unwind_block` must be a cleanup block; otherwise, `unwind_block` must be a terminating block.
```